test/parameter.js

```diff
@@ -135,8 +74,12 @@ describe('Parameter to Member converter', () => {
     const member = parser.convertParameterToMember(parameter);
 
     expect(member.value).to.be.instanceof(minim.elements.Array);
-    expect(member.value.toValue()).to.deep.equal([]);
-    expect(parser.result.toValue()).to.deep.equal(['Value of example should be an array']);
+
+    expect(member.value.length).to.equal(1);
```

Comment:
Because of the example value was being set (and not discarded) in the previous implementation. A further check where we determine if there is no values and set an empty value from the parameter items type wasn't previously handled:

```js
if (parameter.type === 'array' && parameter.items && !example) {
  element.content = [this.convertParameterToElement(parameter.items, true)];
}
```